dist/tools/toolchains/build_gnuarm.sh

```diff
@@ -25,6 +25,9 @@ MAKE_THREADS=-j4
 DOWNLOADER=wget
 DOWNLOADER_OPTS="-nv -c"
 
+MD5=md5sum
```

Comment:
I guess you are targeting OSX which doesn't provide md5sum ?
Then in this case, why not using a conditional case for Darwin and another for Linux using the output of `uname` command ?
See `boards/arduino-due/dist/flash.sh` for an example.